src/main/java/fi/metatavu/metaform/server/rest/RealmsApiImpl.java

```diff
@@ -263,12 +265,18 @@ public Response deleteReply(String realmId, UUID metaformId, UUID replyId) throw
     if (!reply.getMetaform().getId().equals(metaform.getId())) {
       return createNotFound(NOT_FOUND_MESSAGE);
     }
-    
+
     replyController.deleteReply(reply);
     
     return null;
   }
 
+  @Override
+  public Response replyExport(String realmId, UUID metaformId, UUID replyId, String format) throws Exception {
+    // TODO: Add in issue #46 (https://github.com/Metatavu/metaform-api/issues/46)
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.metaform-api%3Ametaform-api&pullRequest=47&issues=AWVk3wFz4Bfu85TPTV1F&open=AWVk3wFz4Bfu85TPTV1F)

[ISSUE_KEY]: AWVk3wFz4Bfu85TPTV1F